### ISSUE TITLE
Microsoft Teams: Consider adding option to mute announcement of incoming notifications when the app is in the background

### ISSUE THREAD

**Issue by elliott94** [nvaccess/nvda#11450]
### Is your feature request related to a problem? Please describe.

When using the Windows Teams app with NVDA, some really useful information is automatically output - including incoming chat messages, and when in a meeting status updates of attendees are also output automatically (somebody new joining the meeting, and somebody muting their microphone to give two examples). Whilst these status updates are really useful to have, NVDA announces these when Teams doesn't have focus and when another application is active - this has the potential to become an issue if, for example, proofreading a document when Teams is in the background; whenever a new notification is received, NVDA will automatically announce its content, interupting the current phrase that's already being spoken.

### Describe the solution you'd like

To resolve this, I can think of two immediate solutions: NVDA could either simply not speak any notifications from Teams when the window doesn't have focus, or offer an option to mute these from within its Settings GUI.

**Comment by leonardder** [comment]
I don't recall receiving the somebody muting their microphone message when focus is outside Teams itself. Do you have the new experimental separate window experience enabled by any chance?
Also, delivering notifications using the Windows 10 action centre is currently in internal testing at Microsoft, so the behavior is likely to change in the near future.

**Comment by elliott94** [comment]
Ah - I apologise; the example that I gave regarding microphone changes was simply an example of a piece of information that I've heard NVDA announcing, so my immediate assumption was that this would also be spoken outside of the application; I'll perform some further testing with this one, so apologies for any confusion - my fault. I should also point out that this is currently being tested with a build of Windows 7 X64.

**Comment by seanbudd** [comment]
Have you submitted this feedback to MS teams?

**Comment by rperez030** [comment]
@elliott94  have you tried disabling report dynamic Content changes (NVDA +5)? If that works, you could creat an app profile for teams that disables that setting. For background notifications such as new chat messages when the teams app is not in focus, I think you can mute notifications for Teams in the notifications center. If you use windows 11, maybe you can do something with the focus settings too.

**Comment by Adriani90** [comment]
This issue can be solved by adjusting the settings in Microsoft Teams or Windows, or by disabling report of notifications or dynamic content in NVDA's settings.
I am closing as works for me.
